Jenkinsfile

```diff
@@ -6,6 +6,7 @@ node {
   govuk.buildProject(
     skipDeployToIntegration: true,
     rubyLintDiff: false,
+    sassLint: false,
```

Comment:
How come we need this now?

Reply:
I was thinking because scss-lint was removed as a dependency, the CI might fail. But it might be okay

Reply:
(Tho it is generally an unnecessary step in CI for this project - as there's no SASS)

Reply:
Looks like its fine to remove!